eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/PaymentOnion.scala

```diff
@@ -490,6 +508,7 @@ object PaymentOnionCodecs {
     .typecase(UInt64(66099), invoiceRoutingInfo)
     .typecase(UInt64(66100), trampolineOnion)
     .typecase(UInt64(5482373484L), keySend)
+    .typecase(UInt64(181324718L), asyncPayment)
```

Comment:
nit: since we're ordering that list by increasing values for readability (when it will start getting big), maybe move that one up above the `keySend` one?

Reply:
Yes, this is completely ok, inside tlv stream we really don't care, so this value is fine.